apps/rush/typings/npm-package-arg/npm-package-arg.d.ts

```diff
@@ -0,0 +1,93 @@
+// Type definitions for npm-package-arg 5.1.2
+// Project: https://github.com/npm/npm-package-arg
+// Definitions by: pgonzal
+
+declare module 'npm-package-arg' {
+  namespace npmPackageArg {
+    interface IResult {
+      /**
+       * Indicates the type of dependency reference.  For example 'version' indicates
+       * a normal SemVer pattern.  See the package README.md for full docs.
+       *
+       * git - a git repository
+       * tag - a tagged version, e.g. "example@latest"
+       * version - A specific version number, e.g. "example@1.2.3"
+       * range - A version range, e.g. "example@2.x"
+       * file - A local .tar.gz, .tar or .tgz file
+       * directory - A local directory
+       * remote - An HTTP url to a .tar.gz, .tar or .tgz file
+       */
+      type: 'git' | 'tag' | 'version' | 'range' | 'file' | 'directory' | 'remote';
+      /**
+       * True for tag, version and range types.
+       */
+      registry: boolean;
+      /**
+       * If known, the "name" field expected in the resulting package.
+       */
+      name: string | null;
+      /**
+       * For scoped NPM packages, the scope name.
+       */
+      scope: string | null;
+
+      escapedName: string | null;
```

Comment:
>escapedName: string | null; [](start = 6, length = 27)

 A version of name escaped to match the npm scoped packages specification. Mostly used when making requests against a registry. When name is null, escapedName will also be null.